Makefile

```diff
@@ -606,7 +606,7 @@ $(INSTALLER):
 	@echo $(FULL_VERSION) > $(VERSION_FILE)
 
 $(VERIFICATION):
-	@mkdir -p $@
+	mv $(INSTALLER) $@
```

Comment:
moving the whole directory doesn't make sense. Once the installer target has been built its results should remain in place even if there is later a make verification<something>

Reply:
And if you want to rely on what the installer target produces you need to have the verification targets depend on the installer taget(s) which populate the installer directory. 
If you do that then you can clone the directory using e.g., cp -rp

Reply:
Every target that VERIFICATION uses depends on the INSTALLER. For example line 758 of the Makefile:
```
$(VERIFICATION).raw: $(BOOT_PART) $(EFI_PART) $(ROOTFS_IMG) $(INITRD_IMG) $(VERIFICATION_IMG) $(CONFIG_IMG) $(PERSIST_IMG) $(BSP_IMX_PART) | $(VERIFICATION)


$(ROOTFS_IMG): $(ROOTFS_TAR) | $(INSTALLER)

$(CONFIG_IMG): $(CONF_FILES) | $(INSTALLER)

etc.
```
So all the pieces required by verification will go to installer dir when you do a `make verification-x` rule. One way to handle it is to copy the rules for these pieces and have them write to the verification directory instead. I chose to reuse the rules and copy everything from the installer's dir.